documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM1.adoc

```diff
@@ -0,0 +1,26 @@
+= Fuse Tooling - What's New in 10.3
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.3.0.AM1
+:page-product_id: jbt_core
+:page-product_version: 4.5.3.AM1
+
+== New shortcuts in Fuse perspective
+
+Shortcuts for Java, Launch and Debug configuration and basic navigation are now enabled by default with Fuse perspective.
+
+The result is a set of actions now available in the Toolbar:
+
+image::./images/newToolbarAction.png[New Toolbar action]
+
+All the associated key shortcuts are also available, such as Ctrl+Shift+T to open a Java Type.
+
+== Performance improvement: loading Advanced tab for Camel components
+
+The load of Advanced tab in Properties view for Camel components has been greatly improved.
+
+image::./images/advancedTab.png[Advanced Tab in Properties view]
+
+In case of Camel Components having a lot of parameters, it was taking several seconds. For instance with the File component, it was taking 3.5s. It is now taking 350ms, response time has been divided by 10. Not ideal but enough to not break anymore user's flow of thought. (see this https://www.nngroup.com/articles/response-times-3-important-limits/[old but interesting article on response time])
```

Comment:
"For Camel Components which have a lot of parameters it was taking several seconds to display the tab contents. For instance the File component was taking ~3.5s to load and show up. After the performance tuning It is now taking ~350ms."

Reply:
I am rather unsure if we should keep the rest. Why should we point users to an article about response times?

Reply:
for motivation which leads to improve it and allow them to learn some concepts if they are not aware with that.

Reply:
imagine you are the user and you find an annoying performance issue. What will you do? Read an article about why its important to have a responsive UI or would you just raise a jira issue so the devs of the tooling can track and fix it?

Reply:
depending on the time that I have at this moment, I will raise a jira and also would be happy to learn some new concept.